Assets/js/app.js

```diff
@@ -0,0 +1,72 @@
+// calling DOM elements
+const addingBook = document.querySelector('.book-list');
+const addBtn = document.querySelector('.add-btn');
+const addTitle = document.querySelector('.title');
+const addAuthor = document.querySelector('.author');
+const form = document.querySelector('.add-book');
+
+// Create a collection that keeps a list of books (hint: you can use an array of objects for that).
+
+let booksCollection = [];
+let j = 0;
+
+function bookLists(item) {
+  return `
+  <li>${item.title}</li>
+  <li>${item.author}</li>
+  <button type='button' id='${item.id}' class='remove-btn'>Remove</button>
+  `;
+}
+
+// Removing book
+
+const removeItem = (ev) => {
+  const removeId = ev.target.id;
+  booksCollection = booksCollection.filter(
+    (x) => x
+      !== booksCollection[
+        booksCollection.findIndex((y) => y.id === parseInt(removeId, 10))
+      ],
+  );
+  localStorage.setItem('bookObject', JSON.stringify(booksCollection));
+  addingBook.innerHTML = `${booksCollection.map(bookLists).join('')}`;
+};
```

Comment:
- [ ] After refreshing the site I can't able to see any book. Please fix that.

Reply:
Done